lib/rules/declaration-block-trailing-semicolon/README.md

```diff
@@ -88,3 +88,21 @@ a { color: pink }
 ```css
 a { background: orange; color: pink }
 ```
+
+## Optional secondary options
+
+### `ignore: ["single-declaration"]`
+
+#### `"single-declaration"`
+
```

Comment:
This heading is duplicate, so it seems needless to me.

```suggestion
```

For example, see the following document:

https://github.com/stylelint/stylelint/blob/67ef0a31db46eb8b6a89e7b47d72c26d827b9395/lib/rules/declaration-block-no-duplicate-properties/README.md#optional-secondary-options

Reply:
Yes, let's remove this heading. We usually only add subheadings when there is more than one possible value for `ignore` and `except` options, [like so](https://github.com/stylelint/stylelint/blob/master/lib/rules/at-rule-empty-line-before/README.md#optional-secondary-options).

(The README `declaration-block-no-duplicate-properties` is out of alignment, in that it should use subheadings.)

Reply:
> We usually only add subheadings when there is more than one possible value for ignore and except options, like so.

I did it exactly for that reason; to prepare such a case.
Ill remove it.